### ISSUE TITLE
"Parts" not accessible from any menu

### ISSUE THREAD

**Issue by skdonnelly** [musescore/MuseScore#23479]
### Issue type

UI bug (incorrect info or interface appearance)

### Steps to reproduce

Documentation refers to "Parts" being in the "File" menu, but it is not there. It *should* be in a menu (like everything) but it should be in the View menu for consistency with the Mixer button which is right beside it in the toolbar and appears in the View menu.

### Screenshots/Screen recordings

![image](https://github.com/musescore/MuseScore/assets/26123297/467d4ab2-89da-4a89-b0db-81bbffd6b6ab)
![image](https://github.com/musescore/MuseScore/assets/26123297/363d5f9c-139f-4fd2-9e89-3557e50602f9)

### MuseScore Version

MuseScore Studio version (64-bit): 4.3.2-241630831, revision: github-musescore-musescore-22b46f2

### Regression

I don't know

### Operating system

OS: Windows 10 Version 2009 or later, Arch.: x86_64,

**Comment by MarcSabatella** [comment]
The documentation shouldn't say that, because indeed, by design, it is no longer there.  Are you sure you are looking at the current version of the Handbook and not an older one?  Can you post a link to where you are seeing that outdated info?

**Comment by cbjeukendrup** [comment]
I think it would make sense to add it back to the file menu, just below project properties.

**Comment by skdonnelly** [comment]
The reference was in the Musescore 3 documentation I realize now, the Musescore 4 documentation is correct.

I still think it is more consistent to have it in the View menu, since the Mixer button corresponds with View > Mixer, the Parts button right beside it should correspond to View > Parts for a consistent user experience. Users shouldn't have to surf through all the menus to find the way to view a specific dialog.

**Comment by cbjeukendrup** [comment]
I'm not a fan of putting it in the View menu, because it has nothing to do with "View". The mixer has a clear reason for being in the View menu, because that menu is about all the side panels and bottom panels that can be opened and closed. The Parts dialog is not such a panel, and fits better together with "Project properties". This should actually _reduce_ the amount of "surfing through all the menus", because it means that options are where they logically belong.

Also, using that top-center toolbar as a guideline for which things need to be located together in the menus makes no sense for two reasons:
- if someone can just use that toolbar, they would do that instead of looking in any menu, so even if the toolbar causes expectations about which things could be in the same menu, those expectations will never have any effect
- currently, that toolbar contains indeed only these two random buttons, but in the future it will include at least a third and probably also a fourth button, and then it will all make more sense

**Comment by MarcSabatella** [comment]
Several other things in the View menu are not side/bottom panels either - Master palette, Instruments, Playback setup.  I'm not saying that's good, though - I agree it's kind of awkward since none of those are about what a View menu is more typically about.  But, none of those are really great candidates for "File" to me, either.

Maybe *all* of these could be moved to a new menu?  Certainly "Instruments" and "Parts" seem like they belong together at least as much as "Project properties" and "Parts" do.  Master palette maybe less so.  I don't know, but it definitely feels like design input would be useful here.

**Comment by skdonnelly** [comment]
I think the Parts dialog could/should just be another palette and be in the View menu with the rest. It's no different from the Instrument properties palette really.

**Comment by loss-of-generality** [comment]
I understand that there is some design input but I would be interested in working on this. Should I wait until there is general consensus on the design or should I go ahead and start working?

**Comment by pacebes** [comment]
Hi

I am looking for a "good first issue". 
I have made some changes to a MuseScore in order to add this "Parts" section just below Project properties". See: 

![PartsInaMenu](https://github.com/user-attachments/assets/09710d78-d83e-4bb6-ad03-5963fd4ccb32)

As far as I understand this change is yet under discussion (if it should be made and where should the option be included) . Am I right ??

Thanks

**Comment by cbjeukendrup** [comment]
@pacebes That's right. If it will be put into the File menu, I would suggest removing the separator line between project properties and parts. 

Also, the title of the menu item should be `Parts…`, with an ellipsis character added to it, just like Project properties. That's because the command opens a dialog, and the convention is to add an ellipsis character to the name such commands.
However, if you add the ellipsis in `notationuiactions.cpp`, then it will also affect the text of the button in the main window, which is not desirable. 
Fortunately, you can solve it by editing only `appmenumodel.cpp`. The `makeMenuItem` method that is inherited from `AbstractMenuModel` namely has a second parameter, which allows specifying the title in the form of a TranslatableString. For example, `TranslatableString("action", "Parts…")`.

After implementing these two suggestions, feel free to open a PR with your changes, even though there is no consensus yet about in which menu the item should appear. After you open a PR, you can still easily modify it, it the designers decide to put the item in a different menu. Thanks in advance!

**Comment by pacebes** [comment]
Thanks @cbjeukendrup 
I will open that PR.

**Comment by pacebes** [comment]
@cbjeukendrup , I have created the pull request: https://github.com/musescore/MuseScore/pull/24039

I don't know if I should set the status to "PR created" or where may I do that:

_ https://musescore.org/es/handbook/developers-handbook/finding-your-way-around/git-workflow#Write_some_code! : 
_If you are fixing an issue from the tracker, set the issue's status to "PR created" with a link to your pull request on GitHub. One of the developers with push rights on the main repo will merge your request ASAP, or leave a comment to say why it hasn't been merged and what needed to be done to get it merged._


Kind regards

**Comment by cbjeukendrup** [comment]
Ah, that's about the old issue tracker that we used to have on MuseScore.org. So no action is needed. 

At some point we should move all remaining useful content of that "developer handbook" to the Wiki here on GitHub, and then archive those pages on MuseScore.org. That's going to be a big task...
